conans/test/unittests/model/build_info/cppflags_test.py

```diff
@@ -9,24 +11,38 @@ class CppFlagsTest(unittest.TestCase):
 
     def test_use_cxxflags(self):
         """ Changes in cxxflags get reflected in cppflags """
-        cpp_info = _CppInfo()
 
-        cpp_info.cxxflags = "flags"
-        self.assertEqual(cpp_info.cppflags, "flags")
-        self.assertEqual(cpp_info.cxxflags, cpp_info.cppflags)
+        with warnings.catch_warnings(record=True) as w:
```

Comment:
``self.assertWarns()`` is not available in python 2, right?

Reply:
It is new in `3.2`.

Nevertheless I think it wouldn't be useful here because I want this function to be executed, so I need some context manager to disable/enable `-Werror`. It is not enough to know that it raises, I need to check that it behaves as expected.